lib/fetching/content-services/facebook-content-service.js

```diff
@@ -92,7 +96,13 @@ FacebookContentService.prototype._doFetch = function(options, callback) {
   }
 
   var isTransient = function(error) {
-    return error.code && _.contains(transientErrorCodes, error.code);
+    var transient;
+    if (error.code) {
+      transient = _.includes(transientErrorCodes, error.code);
+    } else {
+      transient = _.includes(transientErrorMessages, error.message);
+    }
+    return transient;
```

Comment:
So if there is an `error.code` but it's not transient, and the error message is one of the transient ones, we count that as not transient. I think that's fair; we can reconsider when/if those happen.